root/report/DiscogsLinksWithMultipleLabels.js

```diff
@@ -0,0 +1,47 @@
+/*
+ * @flow
+ * Copyright (C) 2018 MetaBrainz Foundation
+ *
+ * This file is part of MusicBrainz, the open internet music database,
+ * and is licensed under the GPL version 2, or (at your option) any
+ * later version: http://www.gnu.org/licenses/gpl-2.0.txt
+ */
+
+import * as React from 'react';
+
+import {withCatalystContext} from '../context';
+import Layout from '../layout';
+import formatUserDate from '../utility/formatUserDate';
+import {l} from '../static/scripts/common/i18n';
+
+import LabelURLList from './components/LabelURLList';
+import FilterLink from './FilterLink';
+import type {ReportDataT, ReportLabelURLT} from './types';
+
+const DiscogsLinksWithMultipleLabels = ({
+  $c,
+  canBeFiltered,
+  filtered,
+  generated,
+  items,
+  pager,
+}: ReportDataT<ReportLabelURLT>) => (
+  <Layout fullWidth title={l('Discogs URLs linked to multiple labels')}>
+    <h1>{l('Discogs URLs linked to multiple labels')}</h1>
```

Comment:
These two lines have been changed to lowercase them (if I wasn't reusing an old commit I would have commited them separately, but reverting to then re-change seemed a bit much).